guides/source/debugging_rails_applications.md

```diff
@@ -219,7 +219,7 @@ irb(main):001:0> Article.pamplemousse
 
 After running `ActiveRecord::Base.verbose_query_logs = true` in the `bin/rails console` session to enable verbose query logs and running the method again, it becomes obvious what single line of code is generating all these discrete database calls:
 
-```
+```ruby
```

Comment:
Because of the mixed content, I think the highlighting here detracts more than it adds.